package.json

```diff
@@ -1,8 +1,16 @@
 {
   "name": "polyfill-service",
+  "version": "0.0.0",
+  "private": true,
   "description": "A polyfill combinator",
-  "main": "lib/index.js",
-  "bin": "bin/polyfill-service",
```

Comment:
Removing the bin, main and npm start script would all count as a breaking change.